### PR TITLE
Added test for QgsMapLayerRegistry

### PR DESCRIPTION
**Pull request by elpaso**
See: PR #2534

### CODE REVIEW

File: tests/src/python/CMakeLists.txt [hunk 1]
```diff
@@ -61,6 +61,8 @@ ADD_PYTHON_TEST(PyQgsVectorColorRamp test_qgsvectorcolorramp.py)
 ADD_PYTHON_TEST(PyQgsVectorFileWriter test_qgsvectorfilewriter.py)
 ADD_PYTHON_TEST(PyQgsVectorLayer test_qgsvectorlayer.py)
 ADD_PYTHON_TEST(PyQgsZonalStatistics test_qgszonalstatistics.py)
+ADD_PYTHON_TEST(PyQgsMapLayerRegistry test_qgsmalayerregistry.py)
```
Comment (SebDieBln) [inline tests/src/python/CMakeLists.txt]: Looks like the typo here is what [failed your Travis CI builds](https://travis-ci.org/qgis/QGIS/jobs/94895415#L844-L846).

Comment (elpaso) [inline tests/src/python/CMakeLists.txt]: @SebDieBln thanks!

File: tests/src/python/test_qgsmaplayerregistry.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+class TestQgsMapLayerRegistry(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_removeMapLayer(self):
+        reg = QgsMapLayerRegistry.instance()
+        reg.removeMapLayers(['not_exists'])
+
```
Comment (luipir) [inline tests/src/python/test_qgsmaplayerregistry.py]: A note... What is the test condition?

Comment (elpaso) [inline tests/src/python/test_qgsmaplayerregistry.py]: Do not segfault

**Comment by m-kuhn** [comment]
Thanks 5a7ee20

**Pull request closed without merging**